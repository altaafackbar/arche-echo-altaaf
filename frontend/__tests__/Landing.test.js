import React from "react";
import renderer from 'react-test-renderer';

import LandingV2 from '../login-pages/Landing-V2';


test('Landing snapShot', async () => {
    const snap = renderer.create(
        <LandingV2 />
    ).toJSON();
    expect(snap).toMatchSnapshot();
});