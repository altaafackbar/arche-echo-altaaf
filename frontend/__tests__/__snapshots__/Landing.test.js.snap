// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing snapShot 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fafafa",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "width": "100%",
      }
    }
  >
    <Image
      source={1}
      style={
        Object {
          "height": 300,
          "width": 300,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "marginTop": "10%",
        "textAlign": "center",
        "width": "100%",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#1f1f1f",
          "fontSize": 24,
          "textAlign": "center",
        }
      }
    >
      Welcome to ARCHE | ECHO!
    </Text>
    <Text
      style={
        Object {
          "color": "#919191",
          "fontSize": 16,
          "padding": 10,
          "textAlign": "center",
        }
      }
    >
      All the resources you need to maximize your child's care.
    </Text>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "justifyContent": "center",
        "padding": 10,
        "width": "100%",
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#8A76B6",
          "borderColor": "transparent",
          "borderRadius": 10,
          "borderWidth": 0,
          "height": 40,
          "justifyContent": "center",
          "opacity": 1,
          "width": "85%",
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "fontSize": 16,
              "fontWeight": "500",
            },
            Object {
              "color": "#fff",
            },
          ]
        }
      >
        Sign In
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "justifyContent": "center",
        "padding": 10,
        "width": "100%",
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "transparent",
          "borderColor": "#8A76B6",
          "borderRadius": 10,
          "borderWidth": 1,
          "height": 40,
          "justifyContent": "center",
          "opacity": 1,
          "width": "85%",
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "fontSize": 16,
              "fontWeight": "500",
            },
            Object {
              "color": "#8A76B6",
            },
          ]
        }
      >
        Sign Up
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "marginTop": "2%",
        "padding": 10,
        "paddingBottom": "10%",
        "width": "90%",
      }
    }
  >
    <View
      accessible={true}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <Text
        onPress={[Function]}
        style={
          Object {
            "color": "#1f1f1f",
            "fontSize": 16,
            "fontStyle": "italic",
            "fontWeight": "500",
          }
        }
      >
        Continue As Guest
      </Text>
    </View>
  </View>
</RCTSafeAreaView>
`;
